openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -802,16 +806,16 @@ In this case, the following keys MUST be present in the response as well:
 
 The Wallet MUST include the most recently received `auth_session` in follow-up requests to the Interactive Authorization Endpoint.
 
-If a wallet receives a `type` value that it does not recognize, it MUST abort the issuance process.
+If a Wallet receives a `type` value that it does not recognize, it MUST abort the issuance process.
 
 Additional keys are defined based on the type of interaction, as shown next.
 
-#### Require Presentation {#iar-require-presentation}
+#### Require Presentation {#iae-require-presentation}
 
 If `type` is set to `openid4vp_presentation`, as shown in the following example, the response MUST further include an `openid4vp_request` parameter containing an OpenID4VP Authorization Request. The contents of the request is the same as for requests passed to the Digital Credentials API (see Appendix A.2 and Appendix A.3 of [@!OpenID4VP]), except as follows: 
 
-* The `response_mode` must be either `iar-post` for unencrypted responses or `iar-post.jwt` for encrypted responses. These modes are used to indicate to the Wallet to return the response back to the same Interactive Authorization Request endpoint URL. 
-* For signed requests, the contents of `expected_origins` MUST contain only the Issuer's Interactive Authorization Request endpoint URL.
+* The `response_mode` must be either `iae-post` for unencrypted responses or `iae-post.jwt` for encrypted responses. These modes are used to indicate to the Wallet to return the response back to the same Interactive Authorization Endpoint. 
```

Comment:
```suggestion
* The `response_mode` must be either `iae_post` for unencrypted responses or `iae_post.jwt` for encrypted responses. These modes are used to indicate to the Wallet to return the response back to the same Interactive Authorization Endpoint. 
```

Nit, but why not keep it consistent with direct_post casing?

Reply:
Ok, updated the PR.